dist/iso.js

```diff
@@ -14,32 +14,70 @@ var parse = function (node, x) {
 };
 
 var Iso = (function () {
-  function Iso() {
+  function Iso(config) {
     _classCallCheck(this, Iso);
 
+    this.markupClassName = config.markupClassName || "___iso-html___";
+    this.markupElement = config.markupElement || "div";
+    this.dataClassName = config.dataClassName || "___iso-state___";
+    this.dataElement = config.dataElement || "div";
     this.html = [];
     this.data = [];
   }
 
-  _prototypeProperties(Iso, {
+  _createClass(Iso, {
+    add: {
+      value: function add(html) {
+        var _state = arguments[1] === undefined ? {} : arguments[1];
+
+        var _meta = arguments[2] === undefined ? {} : arguments[2];
+
+        var state = escapeTextForBrowser(JSON.stringify(_state));
+        var meta = escapeTextForBrowser(JSON.stringify(_meta));
+        this.html.push(html);
+        this.data.push({ state: state, meta: meta });
+        return this;
+      }
+    },
+    render: {
+      value: function render() {
+        var _this = this;
+
+        var markup = this.html.reduce(function (markup, html, i) {
+          return markup + ("<" + _this.markupElement + " class=\"" + _this.markupClassName + "\" data-key=\"" + i + "\">" + html + "</" + _this.markupElement + ">");
+        }, "");
+
+        var data = this.data.reduce(function (nodes, data, i) {
+          var state = data.state;
+          var meta = data.meta;
+
+          return nodes + ("<" + _this.dataElement + " class=\"" + _this.dataClassName + "\" data-key=\"" + i + "\" data-meta=\"" + meta + "\" data-state=\"" + state + "\"></" + _this.dataElement + ">");
+        }, "");
+
+        return "\n" + markup + "\n" + data + "\n";
+      }
+    }
+  }, {
     render: {
       value: function render(html) {
         var state = arguments[1] === undefined ? {} : arguments[1];
         var meta = arguments[2] === undefined ? {} : arguments[2];
+        var config = arguments[3] === undefined ? { markupClassName: "___iso-html___", markupElement: "div", dataClassName: "___iso-state___", dataElement: "div" } : arguments[3];
```

Comment:
we need to actually make sure that if someone just provides a markupClass they also get the default for the markupElement and everything else.